tests/Unit/CssInlinerTest.php

```diff
@@ -3440,10 +3440,7 @@ public function provideInvalidImportRules(): array
                 '@media (max-width: 640px) { p { color: red; } }' . "\n"
                 . '@import "foo.css";',
             ],
-            '@import after incorrectly-cased @charset rule' => [
-                '@CHARSET "UTF-8";' . "\n"
-                . '@import "foo.css";',
-            ],
+            // broken: @import after incorrectly-cased @charset rule
```

Comment:
According to CSS spec, `@charset` should be rejected unless it is entirely lowercase (and followed by exaclty one space, not generic whitespace, then opening double quote).

Reply:
Hmm.  From the specification:

> User agents must ignore any `@import` rule that occurs inside a block or after any non-ignored statement other than an `@charset` or an `@import` rule.

That probably means `@import` after an invalid uppercase `@CHARSET` rule should not be ignored, because the invalid `@CHARSET` rule was ignored and is thus not a "non-ignored statement" - and therefore this test is actually invalid - ?

Reply:
Probably. Then let's disable/change the corresponding test (preferably in a separate PR).